### PR TITLE
#21 Media fields

### LINKED ISSUE #21: Configure File Fields
Add custom fields to audio and image files

### PR DESCRIPTION
**Pull request by brandonhundt**
Closes #21 

- Adds custom fields to WP Media (aka `attachments`)
- Adds license taxonomy

## To Do

- [x] Doesn't appear the custom fields are getting to the API.

## To Review

- [x] Checkout Branch.
- [x] Run `npm run local` (or `npm run refresh` if you want a fresh database.)
- [x] Go to http://the-world-wp.lndo.site/.

> ...then...

- [x] Upload or use an external image file
- [x] Ensure you have an option to "Hide Image" and add a License
- [x] Use an external audio file 
- [ ] Change the audio type to standard audio
- [ ] Ensure the transcript is visible
- [ ] Change the audio type to segment audio
- [ ] Ensure you see way more fields [confirm with mapping spreadsheet](https://docs.google.com/spreadsheets/d/10b6bA1-RHGuNgPtczYhJJWorjS5_kvnsIBscqeJi_Qw/edit#gid=1670198883)
- [ ] Change the audio type to Episode Audio
- [ ] Ensure that you can reference other audio files
- [ ] Confirm API works as intended.

### CODE REVIEW

**Comment by brandonhundt** [comment]
@rpeterman-gp I've run into an issue on this PR. I've added all the fields, but the ACF fields are not appearing in the API response... weird.

Upload an audio file and note how the ACF section is blank. REST is enabled. https://the-world-wp.lndo.site/wp-json/wp/v2/media/35?acf_format=standard

Screen Shot 2022-03-23 at 1 16 57 PM

File: wp-content/themes/newspack-theme/acf-json/group_622a69c137333.json [hunk 1]
```diff
@@ -48,73 +29,111 @@
...
             "type": "taxonomy",
             "instructions": "",
             "required": 0,
-            "conditional_logic": 0,
+            "conditional_logic": [
+                [
+                    {
+                        "field": "field_622a69fd0dfdf",
+                        "operator": "!=",
+                        "value": "program-segment"
```
Comment (rpeterman-gp) [inline wp-content/themes/newspack-theme/acf-json/group_622a69c137333.json:133]: Don't we want this on segment audio? Shouldn't this be `!=` to `standard`?

File: wp-content/themes/newspack-theme/acf-json/group_622a69c137333.json [hunk 2]
```diff
@@ -48,73 +29,111 @@
...
+            "type": "relationship",
             "instructions": "",
             "required": 0,
-            "conditional_logic": 0,
+            "conditional_logic": [
+                [
+                    {
+                        "field": "field_622a69fd0dfdf",
+                        "operator": "==",
+                        "value": "program-episode"
```
[the remark this answers is not in the document]
Comment (brandonhundt) [reply wp-content/themes/newspack-theme/acf-json/group_622a69c137333.json:70]: No it only appears now on Episode audio.

File: wp-content/themes/newspack-theme/acf-json/group_622a69c137333.json (hunk 1 above)
Comment (brandonhundt) [reply wp-content/themes/newspack-theme/acf-json/group_622a69c137333.json]: Fixed!

**Review by rpeterman-gp — APPROVED** [review]
Good work! :shipit:

**Pull request merged**